docs/adrs/001-theming-peer-dependency.md

```diff
@@ -0,0 +1,39 @@
+# ADR-001: Theming peer dependency
+
+2023.04.24
+
+## Context
+
+Garden provides a collection of React components through multiple NPM packages.
+Each package has peerDependencies for `@zendeskgarden/react-theming`, `react`,
+`react-dom`, and `styled-components`. Typically, a package should increase its
+major version when there is a change in its peer dependency requirements.
+However, Garden proposes a DECISION to deviate from this [general
+rule](https://nodejs.org/en/blog/npm/peer-dependencies) and allow special
+treatment for `@zendeskgarden/react-theming` peer dependency.
+
+While this decision conflicts with [semantic versioning](https://semver.org/),
+it benefits both maintainers and consumers by enabling centralized and tested
+hooks and utilities across the codebase. As a result, consumers are not required
+to ingest duplicated and more brittle code. A console warning will notify the
+consumer if a peer version is out-of-date.
+
+Ideally, all Garden package versions should align. However, at the very least,
+the theming peer dependency should align with the highest version of Garden
+component package dependency.
```

Comment:
```suggestion
the theming peer dependency should align with the highest version of a Garden
component package dependency.
```

I don't really have a suggest here but this sentence fragment is worded oddly. Almost like a grammar error. The "correction" I'm suggesting is less a suggestion and moreso how I want to read it. Is that correct, or does it change the intended meaning?